src/plugins/xfer/xfer.c

```diff
@@ -1039,6 +1061,7 @@ xfer_add_cb (const void *pointer, void *data,
     charset_modifier = weechat_infolist_string (infolist, "charset_modifier");
     filename = weechat_infolist_string (infolist, "filename");
     proxy = weechat_infolist_string (infolist, "proxy");
+    token = atoi (weechat_infolist_string (infolist, "token"));
```

Comment:
`token` field should be an integer here:

```C
    token = weechat_infolist_integer (infolist, "token");
```

Reply:
Done.